### ISSUE TITLE
Update 'dotnet octo' tool for .NET Core 3

### ISSUE THREAD

**Issue by cookbr** [OctopusDeploy/OctopusCLI#28]
I'm currently leveraging the tool in a _dotnet/core/sdk:3.0_ container and unfortunately getting the following error ...

> It was not possible to find any compatible framework version
> The specified framework 'Microsoft.NETCore.App', version '2.1.0' was not found.
>   - The following frameworks were found:
>       3.0.1 at [/usr/share/dotnet/shared/Microsoft.NETCore.App]

**Comment by jeremyabbott** [comment]
I had a PR for this in #25 and it got changed in #26, but I think the runtimeconfig.template.json is on the wrong project. I think it needs to be under `/source/Octopus.DotNet.Cli/` and right now its under `/source/Octo/`

**Comment by jeremyabbott** [comment]
Also seems like the docs should be updated to remove the 2.1.300 SDK requirement (https://octopus.com/docs/octopus-rest-api/octo.exe-command-line/install-global-tool), or at least use the latest 2.1 version that has security patches...

**Comment by jeremyabbott** [comment]
> I had a PR for this in #25 and it got changed in #26, but I think the runtimeconfig.template.json is on the wrong project. I think it needs to be under `/source/Octopus.DotNet.Cli/` and right now its under `/source/Octo/`

Even with a runtimeconfig.template.json under /source/Octopus.DotNet.Cli/ the tool fails on debian w/ 3.1 and Alpine w/ 3.1. Branch [here](https://github.com/OctopusDeploy/OctopusCLI/compare/master...jeremyabbott:dotnet-octo-31)

I'm wondering if there's a dependency that's different between 2.1 and 3.1 in this case. I tested against 2.2 on alpine 3.10 and it works as expected. I also installed `fake-cli` within debian/3.1 and alpine/3.1 and it worked, and the project still targets netcoreapp2.1.

**Comment by whereisaaron** [comment]
https://github.com/OctopusDeploy/Issues/issues/5988 claims this is fixed in 6.15 but that doesn't seem to be the case. Octo 7.1 is still unable to work with .Net 3.0.
```
dotnet tool install Octopus.DotNet.Cli --global
You can invoke the tool using the following command: dotnet-octo
Tool 'octopus.dotnet.cli' (version '7.1.0') was successfully installed.

[Container] 2020/02/24 05:55:03 Running command dotnet octo --version
It was not possible to find any compatible framework version
The specified framework 'Microsoft.NETCore.App', version '2.1.0' was not found.
  - The following frameworks were found:
      3.0.0 at [/usr/share/dotnet/shared/Microsoft.NETCore.App]
```

**Comment by matt-richardson** [comment]
This has now shipped in `7.1.2`. Apologies for the inconvenience - we thought we had it sorted.
